component/service/src/main/java/io/meeds/social/security/rest/LoginRest.java

```diff
@@ -0,0 +1,763 @@
+
+  public static final String            ADMINISTRATORS_GROUP       = "/platform/administrators";
+
+  public static final String             USERS_GROUP                = "/platform/users";
+
+  public static final String             EXTERNAL_USERS_GROUP       = "/platform/externals";
+
+  public static final String            MEMBER                     = "member";
+
+  public static final String             LOCATION_HEADER            = "Location";
+
+  public static final UserFieldValidator PASSWORD_VALIDATOR =
+      new UserFieldValidator("password", false, false, 8, 255);
+
+  public static final UserFieldValidator EMAIL_VALIDATOR = new UserFieldValidator("email", false, false);
+
+  public static final UserFieldValidator LASTNAME_VALIDATOR                  =
+      new UserFieldValidator("lastName", false, true);
+
+  public static final UserFieldValidator FIRSTNAME_VALIDATOR                 =
+      new UserFieldValidator("firstName", false, true);
+
+  private static final Log LOG = ExoLogger.getLogger(LoginRest.class);
+
+  @Autowired
+  private PasswordRecoveryService passwordRecoveryService;
+
+  @Autowired
+  private OrganizationService organizationService;
+
+  @Autowired
+  private RegisterUIParamsExtension registerUIParamsExtension;
+
+  @Autowired
+  private SecuritySettingService securitySettingService;
+
+  @Autowired
+  private RemindPasswordTokenService remindPasswordTokenService;
+
+  @GetMapping(path = "/verify")
+  @Produces(MediaType.APPLICATION_JSON)
+  @Operation(summary = "Verify user token", method = "GET")
+  @ApiResponses(value = {
+      @ApiResponse(responseCode = "200", description = "Request fulfilled"),
+  })
+  public ResponseEntity<String> verifyToken(HttpServletRequest request,
+                                            @Parameter(description = "Token")
+                                            @RequestParam("token")
+                                            String token,
+                                            @Parameter(description = "Token Type")
+                                            @RequestParam(name = "tokenType", required = false, defaultValue = "")
+                                            String tokenType) {
+
+    String username = passwordRecoveryService.verifyToken(token, tokenType);
+    if (username != null) {
+      return ResponseEntity.ok().body(new JSONObject()
+                                          .put("username", username).toString());
+    } else {
+      return ResponseEntity.notFound().build();
+    }
+  }
+
+  @PostMapping(path = "/finishRegistration")
+  @Produces(MediaType.APPLICATION_JSON)
+  @Operation(summary = "Finalize external user registration", method = "POST")
+  @ApiResponses(value = {
+      @ApiResponse(responseCode = "200", description = "Request fulfilled"),
+  })
+  public ResponseEntity<String> finishRegistration(HttpServletRequest request,
+                                            @Parameter(description = "Token")
```

Comment:
Code format

Reply:
done

Reply:
It appears not yet in fact.

```suggestion
  public ResponseEntity<String> finishRegistration(HttpServletRequest request,
                                                                                          @Parameter(description = "Token")
```

Reply:
Seems ok now on my side :
<img width="2201" height="463" alt="image" src="https://github.com/user-attachments/assets/0bdee626-ad66-4d09-91a3-59b215199c67" />

Can you recheck ?

Reply:
ok, may be Github doesn't display well spaces, strange because the `verifyToken` is well displayed.
I mark this as resolved.